libraries/WiFi/src/ssl_drv.h

```diff
@@ -0,0 +1,33 @@
+/* Provide SSL/TLS functions to ESP32 with Arduino IDE
+ * Ported from Ameba8195 (no license specified) to ESP32 by Evandro Copercini - 2017 - Public domain
```

Comment:
I'm afraid we need to get some confirmation about the original license of these files before we can consider adding them to the project.